hyperframe/frame.py

```diff
@@ -103,6 +107,33 @@ def parse_body(self, data):
         raise NotImplementedError()
 
 
+Flag = collections.namedtuple("Flag", ["name", "bit"])
+
+
+class Flags(MutableSet):
```

Comment:
Good point. On another note, shall we keep Flags in `frame.py` or do we want to move it to `flags.py`?